### PR TITLE
Modularical

### PR DESCRIPTION
**Pull request by zysim**
Closes https://github.com/dshepsis/OkamiMap/issues/4

Basically, I moved the main script into its own `index.js`, and within that, broke off parts of the code into their own modules. Tbh a bit of the breaking off I've done is done by feeling, and not because I've got a rigid plan in mind.

Read notes for extra details :)

### CODE REVIEW

File: js/zoomViewer.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+let zoomViewer
+
+export default map => {
+  if (zoomViewer != null) return zoomViewer
```
Comment (zysim) [inline js/zoomViewer.js:4]: Honestly I'm not sure if I should have it return an instance of the Zoom Viewer (with it already added to the map), which I've done here, or if I should return the constructor.

File: js/index.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import zoomViewer from './zoomViewer.js'
+import mapGen from './mapGen.js'
+import paraFetchJSON from './paraFetchJson.js'
+import addImageOverlayToMap from './addImageOverlayToMap.js'
```
Comment (zysim) [inline js/index.js:4]: Feel free to come up with a more systematic terminology for naming modules

File: js/zoomViewer.js (hunk 1 above)
Comment (dshepsis) [reply js/zoomViewer.js:4]: Based on https://leafletjs.com/examples/extending/extending-3-controls.html and https://github.com/Leaflet/Leaflet.fullscreen/blob/gh-pages/dist/Leaflet.fullscreen.js, I think the pattern for leaflet controls should be to import the leaflet object and attach things to it. In this case, we'd modify `L.Control` and `L.control`.

**Review by dshepsis — APPROVED** [review]
Simply extracts the css into a css file and the js into a js file. Seems like a good basic step.

**Pull request merged**